manifest.tmpl

```diff
@@ -1,10 +1,5 @@
 image: rancher/hardened-flannel:{{build.tag}}
 manifests:
-  -
-    image: rancher/hardened-flannel:{{build.tag}}-amd64
-    platform:
-      architecture: amd64
-      os: linux
```

Comment:
We still need to push the manifest in some way, one of the challenges of this hybrid approach is that there isn't a straightforward way of making the manifest in two different places.
One way of doing it is creating the manifest just on drone, or just on gh actions. The issue with this approach is that it may fail if the image built on the other CI system wasn't pushed yet.
Other way is using docker buildx to build for ARM in gh actions and drop drone completely.

Reply:
More context: The way we do multi-arch images now is to push an image with an arch suffix, like `rancher/hardened-flannel:v0.24.3-build20240307-amd64` and `rancher/hardened-flannel:v0.24.3-build20240307-arm64` then, after those two images are in the registry, we push a manifest that creates a new tag `rancher/hardened-flannel:v0.24.3-build20240307` that points to the two previously pushed tags.
I couldn't find a way of appending information to an existing manifest

Reply:
yeah, I'm waiting on Matt to tell me if it is find to use docker buildx for ARM64

Reply:
in fact we do it in upstream flannel and it works fine: https://github.com/flannel-io/flannel/blob/8ac2e92beb6255243d39dc6dfb2690ca38c483fb/.github/workflows/release.yml#L55

Reply:
yes, the PR is now changed. I'm using buildx to do multi-arch build